CHANGELOG.md

```diff
@@ -5,6 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [0.86.0] - 2021-07-28
```

Comment:
We've fallen out of sync again - 0.86.0 has already been released.

Would you like me to look into this as part of my support day?